.github/workflows/deploy-to-cloud-run.yml

```
@@ -0,0 +1,255 @@
+        mkdir -p ../backend/templates
+        cp -r out/* ../backend/templates/
+        
+        echo "✅ Frontend build completed"
+        echo "📊 Templates directory contents:"
+        ls -la ../backend/templates/
+
+    - name: Setup Google Cloud CLI
+      uses: google-github-actions/setup-gcloud@v2
+      with:
+        project_id: price-comparison-app-463007
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
```

actionlint (1.7.7)

input "service_account_key" is not defined in action "google-github-actions/setup-gcloud@v2". available inputs are "install_components", "project_id", "skip_install", "version"

(action)